data/ucr-program-participation.json

```diff
@@ -0,0 +1,52 @@
+{
+  "alabama": {
+    "srs": true,
+    "nibrs": false,
+    "hybrid": false
+  },
+  "alaska": {
+    "srs": false,
+    "nibrs": false,
+    "hybrid": true
+  },
+  "arizona": {
+    "srs": false,
+    "nibrs": false,
+    "hybrid": true
+  },
+  "arkansas": {
+    "srs": false,
+    "nibrs": true,
+    "hybrid": false
+  },
+  "california": {
+    "srs": true,
+    "nibrs": true,
+    "hybrid": true
+  },
+  "colorado": {
+    "srs": false,
+    "nibrs": true,
+    "hybrid": false
+  },
+  "connecticut": {
+    "srs": false,
+    "nibrs": false,
+    "hybrid": true
+  },
+  "delaware": {
+    "srs": false,
+    "nibrs": true,
+    "hybrid": false
+  },
+  "idaho": {
+    "srs": false,
+    "nibrs": true,
+    "hybrid": false
+  },
+  "virginia": {
+    "srs": true,
+    "nibrs": false,
+    "hybrid": false
+  }
```

Comment:
I need to add the rest of the states to this data file. I'll do that by pulling information from https://docs.google.com/spreadsheets/u/1/d/1APcLiophXeeDrkhjMCULq1NJR1rbhrdkRPUyzV2OT_0/edit#gid=0